js/wizard/wizardTabGeneric.js

```diff
@@ -381,6 +422,28 @@ OCA = OCA || {};
 			}
 			this.configModel.set($element.attr('id'), value);
 		},
+		/**
+		 * queue up al changed elements from the model
+		 * represented by a HTML element and its ID.
+		 *
+		 * @param {jQuery|viewSaveInfo} $element
+		 * @private
+		 */
+		_queueChanges: function($element) {
+			this.saveQueue.push($element);
+			this._enableButton('.ldap_action_save');
+		},
+
+		_saveQueuedChanges: function () {
+			var self = this;
+			var saveQueue = _.uniq(this.saveQueue);
+
+			_.forEach(saveQueue, function(value) {
+				self._requestSave(value);
```

Comment:
It get's reenabled in the current state.
This call does $.post() down the line.